build.gradle

```diff
@@ -1416,14 +1416,6 @@ project(':streams') {
   dependencies {
     api project(':clients')
 
-    // use `api` dependency for `connect-json` for compatibility (e.g. users who use `JsonSerializer`/`JsonDeserializer`
-    // at compile-time without an explicit dependency on `connect-json`)
-    // this dependency should be removed after we unify data API
-    api(project(':connect:json')) {
-      // this transitive dependency is not used in Streams, and it breaks SBT builds
-      exclude module: 'javax.ws.rs-api'
-    }
-
     implementation libs.slf4jApi
     implementation libs.rocksDBJni
     implementation libs.jacksonAnnotations
```

Comment:
"implementation libs.jacksonDatabind" is already being added to compileClassPath in streams module.
Thus it seems there indeed there's no need to add compile dependency of libs.jacksonDatabind to streams module. Please let me know if this is not the case.

Reply:
SGTM.

Back in the days, KafkaStreams did not use anything JSON related, but in a later version we actually added this dependency.